include/fi_rbuf.h

```diff
@@ -226,16 +227,15 @@ static inline int rbfdinit(struct ringbuffd *rbfd, size_t size)
 	if (ret < 0)
 		goto err1;
 
-	flags = fcntl(rbfd->fd[RB_READ_FD], F_GETFL, 0);
-	ret = fcntl(rbfd->fd[RB_READ_FD], F_SETFL, flags | O_NONBLOCK);
-	if (ret < 0)
+	ret = fi_fd_nonblock(rbfd->fd[RB_READ_FD]);
+	if (ret)
```

Comment:
no ideas. i just replaced 2*fcntl by call to existing function :)
but you are right, fi_fd_... is out of naming convention